src/routes/messages.js

```diff
@@ -1,11 +1,47 @@
 module.exports = (() => {
 	const express = require('express');
 	const router = express.Router();
-	// const path = require('path');
-	// const messageController = require(path.join(__dirname, '../controllers/message.controller'));
+	const path = require('path');
+	const messageController = require(path.join(__dirname, '../controllers/message.controller'));
 
```

Comment:
I'll need to modify the membership middleware just slightly to check based on mid (or cid) whether the user is part of the chat (I just did that real quick), so I can plug it in today once this PR is updated and merged. 🙂